pilot/helpers/Project.py

```diff
@@ -233,11 +237,13 @@ def finish(self):
             self.previous_features = get_features_by_app_id(self.args['app_id'])
             if not self.skip_steps:
                 print('', type='verbose', category='pythagora')
+                if self.run_command and self.check_ipc():
+                    print(self.run_command, type='run_command')
                 feature_description = ask_user(self, "Project is finished! Do you want to add any features or changes? "
                                                      "If yes, describe it here and if no, just press ENTER",
                                                require_some_input=False)
 
-                if feature_description == '':
+                if feature_description == '' or feature_description == 'continue':
```

Comment:
can there ever be a "continue" button here?

Reply:
not sure why but when there is `run_command` extension shows `Continue` button 🤷‍♂️  I guess they come in pair...